React/tic-tac/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=http://localhost:3001/api/
```

Comment:
This file shouldn't be commited. Contains sensible data. Add it to `.gitignore` file